crates/xilem_core/src/any_view.rs

```diff
@@ -3,22 +3,22 @@
 
 #[macro_export]
 macro_rules! generate_anyview_trait {
-    ($anyview:ident, $viewtrait:ident, $viewmarker:ty, $cx:ty, $changeflags:ty, $anywidget:ident, $boxedview:ident; $($ss:tt)*) => {
+    ($anyview:ident, $viewtrait:ident, $viewmarker:ty, $cx:ty, $changeflags:ty, $anywidget:ident, $boxedview:ident; ($($super_bounds:tt)*), ($($state_bounds:tt)*)) => {
```

Comment:
I was going to suggest changing the input to avoid the parentheses here too, but unfortunately that runs into https://github.com/rust-lang/rust/issues/18700 (last comment has a workaround, but probably not worth using that here).

Reply:
Maybe make the last two parameters optional using `$()?` though? The invocation with `(), ()` looks weird.

Reply:
I've used optionals as you said